force-app/main/default/lwc/potentialDuplicates/potentialDuplicates.html

```diff
@@ -0,0 +1,19 @@
+<template>
+    <template lwc:if={isEnabled}>
```

Comment:
Replacing `isEnabled` with `displayCard` (or setting it to the same value) should hide the component and allow the Toast message to still display.